### DISCUSSION TITLE
Can't install next-contentlayer in nextjs 14.1.0, how to fix?

### DISCUSSION THREAD

**Discussion by mdranaa** [timlrx/tailwind-nextjs-starter-blog#852] [General]
When I am trying to install next-contentlayer in nextjs 14.1.0. It's showing error. I can't install next-contentlayer.
Can anyone suggest me how can I install it?

**Comment by leonliu06** [comment]
I just retry with `--legacy-peer-deps`, namely, `npm install --legacy-peer-deps`.
